saleor/tests/e2e/orders/test_order_mark_as_paid.py

```diff
@@ -0,0 +1,84 @@
+import pytest
+
+from .. import DEFAULT_ADDRESS
+from ..orders.utils.draft_order_complete import draft_order_complete
+from ..orders.utils.draft_order_create import draft_order_create
+from ..orders.utils.draft_order_update import draft_order_update
+from ..orders.utils.order_lines_create import order_lines_create
+from ..orders.utils.order_mark_as_paid import mark_order_paid
```

Comment:
You can change it to

from .utils import (
    draft_order_complete,
    draft_order_create,
    draft_order_update,
    order_lines_create,
    mark_order_paid,
)